src/App.js

```diff
@@ -1,25 +1,77 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import contactsArray from "./contacts.json";
+import { useState } from "react";
 
 function App() {
+  const contacts = [
+    contactsArray[0],
+    contactsArray[1],
+    contactsArray[2],
+    contactsArray[3],
+    contactsArray[4],
+  ];
+  const [actors, setActors] = useState(contacts);
+
+  const deleteContact = (contactId) => {
+    setActors((prevActors) => {
+      const newList = prevActors.filter((element) => {
+        return element.id !== contactId;
+      });
+      return newList;
+    });
+  };
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <button
+        onClick={() => {
+          setActors((prevActors) => {
+            let randomIndex = Math.floor(Math.random() * contactsArray.length); // random index
+            let randomContact = contactsArray[randomIndex]; // pick a random from whole list
+            prevActors.push(randomContact); // push to newList
```

Comment:
you can use push method whenever you have shallow copy, because it modifies directly your array,
here you can use this syntax to update your state :
 const newArr = [...prevActors , randomContact]
and return newArr